### DISCUSSION TITLE
[Kavita]  Problems setting up email

### DISCUSSION THREAD

**Discussion by Pherscho** [Kareadita/Kavita#2736] [Ideas]
### Idea Description

I am not sure if this is the correct place to ask, but I am having problems setting up email.  I need emailing functionality to be able to send books to my kindle.   When I try to setup the email using GMAIL I get a small red window saying there is a wrong second factor authentication.   I see nothing in the logs.   I've been stuck in this for long now.  Any ideas?

### Idea Category

API

### Duration of Using Kavita

1 day

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is not the correct place. Please create an ISSUE and I can support you there. This is for feature requests only.
